internal/pkg/mapper/defaultbucket/defaultbucket_test.go

```diff
@@ -0,0 +1,48 @@
+package defaultbucket_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem/aferofs"
+	"github.com/keboola/keboola-as-code/internal/pkg/mapper"
+	"github.com/keboola/keboola-as-code/internal/pkg/mapper/defaultbucket"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/testapi"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func createMapper(t *testing.T) (*mapper.Mapper, model.MapperContext, *utils.Writer) {
+	t.Helper()
+	logger, logs := utils.NewDebugLogger()
+	fs, err := aferofs.NewMemoryFs(logger, ".")
```

Comment:
Na to existuje aj skratka `testhelper.NewMemoryFs` , nevracia err.

Reply:
👍